ingest/validation/ontologies/version.txt

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 1737653567 # validation cache key
+=======
+1738072997 # validation cache key
+>>>>>>> 3262903a4d13e2ecacd8cac9b39f34f2449bc119
```

Comment:
Committing the diff seems like it could cause a problem in CI, and perhaps cause clients to refetch ontologies on every upload UI page load.

```suggestion
<<<<<<< HEAD
1737653567 # validation cache key
=======
1738072997 # validation cache key
>>>>>>> 3262903a4d13e2ecacd8cac9b39f34f2449bc119
```

Reply:
Ah, thanks for catching that. The rest of the ontology files were compressed so they couldn't be merged and  I missed that this file needed a merge conflict resolution. Fixed in `c6872ca`